locale/es/manageiq.po

```diff
@@ -10606,9 +10609,15 @@ msgstr "ConfigurationScriptBase|Salida"
 msgid "ConfigurationScriptBase|Payload"
 msgstr "ConfigurationScriptBase|Carga útil"
 
+msgid "ConfigurationScriptBase|Payload error"
+msgstr "ConfigurationScriptBase | Error de carga útil"
+
 msgid "ConfigurationScriptBase|Payload type"
 msgstr "ConfigurationScriptBase|Tipo de carga útil"
 
+msgid "ConfigurationScriptBase|Payload valid"
+msgstr "ConfigurationScriptBase | Carga útil válida"
```

Comment:
space around pipe

Reply:
will need to make sure the translators get this change we did manually, along with the others.